src/snowcli/cli/snowpark/procedure/commands.py

```diff
@@ -49,6 +53,51 @@
 )
 app.add_typer(procedure_coverage_app)
 
+FileOption = typer.Option(
+    "app.zip",
+    "--file",
+    "-f",
+    help="Path to the file or folder to containing the procedure code. If you specify a directory, the procedure deploys the procedure in the default `app.zip` file.",
+    exists=False,
+)
+
+IdentifierOption = typer.Argument(
+    ...,
+    help="String containing the procedure signature with its inputs, such as 'hello(int, string)'.",
+)
```

Comment:
I'm wondering if we should abstract the `Identifier` argument? Maybe we could create it from two inputs: object name and optional example? WDYT?